tools/openms/OpenSwathWorkflow.xml

```diff
@@ -1,598 +1,480 @@
+        <param name="use_rt_score" argument="-Scoring:Scores:use_rt_score" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="true" label="Use the retention time score (this score measure the difference in retention time)" help=""/>
+        <param name="use_library_score" argument="-Scoring:Scores:use_library_score" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="true" label="Use the library score" help=""/>
+        <param name="use_intensity_score" argument="-Scoring:Scores:use_intensity_score" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="true" label="Use the intensity score" help=""/>
+        <param name="use_nr_peaks_score" argument="-Scoring:Scores:use_nr_peaks_score" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="true" label="Use the number of peaks score" help=""/>
+        <param name="use_total_xic_score" argument="-Scoring:Scores:use_total_xic_score" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="true" label="Use the total XIC score" help=""/>
+        <param name="use_total_mi_score" argument="-Scoring:Scores:use_total_mi_score" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="false" label="Use the total MI score" help=""/>
+        <param name="use_sn_score" argument="-Scoring:Scores:use_sn_score" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="true" label="Use the SN (signal to noise) score" help=""/>
+        <param name="use_mi_score" argument="-Scoring:Scores:use_mi_score" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="false" label="Use the MI (mutual information) score" help=""/>
+        <param name="use_dia_scores" argument="-Scoring:Scores:use_dia_scores" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="true" label="Use the DIA (SWATH) scores" help="If turned off, will not use fragment ion spectra for scoring"/>
+        <param name="use_ms1_correlation" argument="-Scoring:Scores:use_ms1_correlation" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="false" label="Use the correlation scores with the MS1 elution profiles" help=""/>
+        <param name="use_sonar_scores" argument="-Scoring:Scores:use_sonar_scores" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="false" label="Use the scores for SONAR scans (scanning swath)" help=""/>
+        <param name="use_ion_mobility_scores" argument="-Scoring:Scores:use_ion_mobility_scores" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="false" label="Use the scores for Ion Mobility scans" help=""/>
```

Comment:
@bernt-matthias boolean should not be optional I think.

Reply:
True.